### DISCUSSION TITLE
安卓字体和字重问题。

### DISCUSSION THREAD

**Discussion by cwjTerrace** [youzan/vant#10175] [Q&A]
font-weight-bold: 500 这个属性在2022年安卓兼容性如何？是否有调研？

**Comment by cwjTerrace** [comment]
我看vant用的默认字重是@font-weight-bold: 500;

**Comment by cwjTerrace** [comment]
@chenjiahan

**Comment by chenjiahan** [comment]
这个看系统字体的，如果字体本身不支持 500 的话，会变成 bold。

**Reply by cwjTerrace** [reply]
![middle_img_v2_3ac55090-fa4a-4865-9a3d-3a598944fedg](https://user-images.githubusercontent.com/29698222/149443223-6b78a42e-e0fd-4280-89df-df986b0df17f.jpg)
这是我测试渲染字重的[地址](https://fairyland.run/public-pages/font-weight.html)的截图
500会渲染成400
